rust/src/treefile.rs

```diff
@@ -22,12 +22,18 @@
 
 use serde_json;
 use serde_yaml;
-
+use std::io::prelude::*;
 use std::path::Path;
 use std::{fs, io};
+use tempfile;
 
 const ARCH_X86_64: &'static str = "x86_64";
 
+pub struct Treefile {
+    pub json_fd: fs::File,
```

Comment:
Given that this will be retrieved only once and that it could go completely unused in the `--enable-rust` path once we add accessors, let's just do this at `rpmostree_rs_treefile_get_json_fd` time?

Reply:
OK, done :arrow_down: